index.html

```diff
@@ -12,40 +12,235 @@
+                    class="fa-brands fa-twitter"></i></a></li>
+              <li class="social-icon-item"><a href="https://www.linkedin.com/in/belay-birhanu-144ba714b/" aria-label="Linkedin" target="_blank"
+                  rel="noopener"><i class="fa-brands fa-linkedin-in"></i></a></li>
+              <li class="social-icon-item"><a href="https://github.com/belaymit?tab=repositories" aria-label="Github" target="_blank" rel="noopener"><i
+                    class="fa fa-github"></i></a></li>
+              <li class="social-icon-item"><a href="#" aria-label="Angellist" target="_blank" rel="noopener"><i class="fa-brands fa-angellist"></i></a>
+              </li>
+            </ul>
+          </div>
+          <button>Get my resume</button>
+          <div class="languages">
+            <ul>
+              <li class="language-item">
+                <h1>Languages</h1>
+                <a href="#"><svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z"
+                    fill="#5E6C84" />
+                </svg></a>
+              </li>
+              <div class="language">
+                <div class="language-icon">
+                  <img src="./Images/Ellipse 1.png" alt="">
+                </div>
+                <span>JavaScript</span>
+              </div>
+              <div class="language">
+                <div class="language-icon">
+                  <img src="./Images/Ellipse_2.png" alt="">
+                </div>
+                <span>HTML</span>
+              </div>
+              <div class="language">
+                <div class="language-icon">
+                  <img src="./Images/Ellipse_3.png" alt="">
+                </div>
+                <span>CSS</span>
+              </div>
+              <li class="language-item framework-skills">
+                <h1>Frameworks</h1>
+                <a href="#"><svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z"
+                    fill="#505F79" />
+                </svg></a>
+              </li>

```

Comment:
Kindly follow the margins defined in the Figma design

### Your design

<img width="302" alt="image" src="https://user-images.githubusercontent.com/78151836/198693294-6892fe8a-8b4e-459c-90e1-8a22de5ced0a.png">


### Figma design

<img width="311" alt="image" src="https://user-images.githubusercontent.com/78151836/198694612-9901dd17-b649-439d-9f07-35edba6607b2.png">